composer.json

```
@@ -14,13 +14,13 @@
     "license": "MIT",
     "require": {
         "php": "^8.2",
-        "illuminate/contracts": "^11.0",
-        "illuminate/support": "^11.0"
+        "illuminate/contracts": "^12.0",
+        "illuminate/support": "^12.0"
```

This should support **both** versions, otherwise it is considered a BC and will require a major version bump:

```suggestion
        "illuminate/contracts": "^11.0 || ^12.0",
        "illuminate/support": "^11.0 || ^12.0"
```